sdk/js/src/token_bridge/createWrapped.ts

```diff
@@ -164,3 +166,59 @@ export function createWrappedOnAptos(
 ): Types.EntryFunctionPayload {
   return createWrappedCoinAptos(tokenBridgeAddress, attestVAA);
 }
+
+export async function createWrappedOnSuiPrepare(
+  network: Network,
+  coreBridgeAddress: string,
+  tokenBridgeAddress: string,
+  attestVAA: Uint8Array,
+  signerAddress: string
+): Promise<TransactionBlock> {
+  return publishCoin(
+    network,
+    coreBridgeAddress,
+    tokenBridgeAddress,
+    uint8ArrayToHex(attestVAA),
+    signerAddress
+  );
+}
+
+export async function createWrappedOnSui(
+  provider: JsonRpcProvider,
+  tokenBridgeAddress: string,
+  coreBridgeStateObjectId: string,
+  tokenBridgeStateObjectId: string,
+  signerAddress: string,
+  coinType: string
```

Comment:
Made a comment in _sui-integration.ts_ about this, but we may want to pass in the coin package ID and generate the coin type within this method.

Reply:
Hmm I think type would be just as easy to pass in because they could get it from the `WrappedAssetSetup` type vs getting the bare `packageId` from the `Coin` publish event, but I don't have strong opinions, so I'll make the change